Sources/MapboxMaps/Ornaments/ScaleBar/MapboxScaleBarOrnamentView.swift

```diff
@@ -180,9 +184,7 @@ internal class MapboxScaleBarOrnamentView: UIView {
         let halfLabelWidth = ceil(lastLabelWidth / 2)
 
         size = CGSize(width: totalBarWidth + halfLabelWidth, height: 16)
```

Comment:
So this is modifying size and calling setNeedsLayout. Although we're not calling updateConstraints, `intrinsicContentSize` is returning size - so I wonder what the behavior is if there's a view to the side of the scale bar.

How does the scale bar look when it's on the right hand side of the view?

Reply:
I am working on resolving some issues when the scale bar is on the top right. Currently only seeing the `0`. If I restore `updateScaleBar` to `updateConstraints`, I see the scaleBar but `layoutSubviews` is called repeatedly.